### PR TITLE
Update Pawket related information.

### PR DESCRIPTION
**Pull request by wizicer**
1. Update Pawket link and open source status
2. Update link and description of Pawket.
3. Add air-gapped column to wallet comparison table.
  image

Please let me know if any inappropriate content found.

### CODE REVIEW

**Comment by SlowestTimelord** [comment]
Thanks for the update @wizicer! I think we should agree on what are the main useful features to compare wallets on in the table, and which to supply as supplementary information. For example `memo` may not be an important feature that could be replaced by `air gapped`. My concern is the table getting too cluttered due to limitations of the current markdown table design.

@sargonas any thoughts?

**Comment by SlowestTimelord** [comment]
Another useful (recently relevant in light of Nucle news) feature comparison column are 12- and 24-word support, of which Pawket and Chia reference wallet can handle both.

**Comment by wizicer** [comment]
Just removed memo, it looks like:
image

IMHO: 12/24-word support is not that important (compares to current 5 features) to user.

**Comment by sargonas** [comment]
I would argue 12/24 word support is still relevant for a short while longer, but will likely become irrelevant sooner than later. However I have no strong opinion one way or another on if they should be included or removed.

I agree in the removal of Memo support simply because how the system does "memos" is not how people expect/think it does and there are various ways in which it is used and a simple binary comparison is misleading.

Air-Gapped seemed a bit interesting, but can you tell me a bit more about how Pawket is Air-Gapped @wizicer? Just curious because it seems like the only way to get it onto a mobile device would be to download it onto a device, which means the device has to have at _some_ point or another network connectivity. Even if you disable it after doing so, by definition the device the wallet is on is no longer considered to be air-gapped so it's unclear to me how it can be an air-gapped wallet, though I gladly welcome being corrected if I am missing something.

I also think there should be a trinary field for "Actively Maintained" with a Solid Yes that is is under active support, a solid no that it is not, and a question mark when it is unclear.

Lastly I'm putting in a PR in just a second to rename "Chia Official" to "Chia Reference"

**Comment by SlowestTimelord** [comment]
> I would argue 12/24 word support is still relevant for a short while longer, but will likely become irrelevant sooner than later. However I have no strong opinion one way or another on if they should be included or removed.

Instead of having it here, probably a good addition to the FAQ update as part of #7 

> I agree in the removal of Memo support simply because how the system does "memos" is not how people expect/think it does and there are various ways in which it is used and a simple binary comparison is misleading.

Agreed

> Air-Gapped seemed a bit interesting, but can you tell me a bit more about how Pawket is Air-Gapped @wizicer? Just curious because it seems like the only way to get it onto a mobile device would be to download it onto a device, which means the device has to have at _some_ point or another network connectivity. Even if you disable it after doing so, by definition the device the wallet is on is no longer considered to be air-gapped so it's unclear to me how it can be an air-gapped wallet, though I gladly welcome being corrected if I am missing something.

Perhaps "offline" is a better descriptor, but I also don't understand what its capabilities are offline that other existing wallets don't support. For example I could create offers with the Chia Reference wallet offline as long as it is synced to a certain point where the coin id exists?

> I also think there should be a trinary field for "Actively Maintained" with a Solid Yes that is is under active support, a solid no that it is not, and a question mark when it is unclear.

Do you have a sense of which wallets should have which status?

> Lastly I'm putting in a PR in just a second to rename "Chia Official" to "Chia Reference"

Great call, the links.yml entry itself should also be updated.

**Comment by wizicer** [comment]
Let explain a little more about how I utilized air-gapped Pawket:

1. I refreshed one Android phone, and set WiFi/Bluetooth/GPS disabled.
2. With USB, upload `pawket.apk` to the phone and install. (no internet connection needed) And so far an air-gapped Pawket has been created (let's call this AGP for short).
3. Create account with AGP, share account public key* with an Online Pawket(let's call this ONP for short).
4. On ONP, add public key account in account list. Then you will get a full functional account except signing.
5. You can view balance, send XCH/CAT, batch send, issue CAT, mint NFT and almost everything just like it's online. However, until you want sign the transaction, you will be prompted with one or more QR code to sign.
6. You must use AGP to scan the QR code on ONP to generate signature, and then use ONP to scan the signature QR code on AGP. Finally ONP put the signature into the spendbundle and send to the blockchain.

*: to get account public key, click `Scan` -> `Public Key QR Code` in the logged-in UI.

The distinctive Air-gapped Pawket stand out from other wallet as it not only support you to store assets in cold storage, but also let you use it conveniently like using a hardware wallet.

From google, first definition from NIST is [An interface between two systems at which (a) they are not connected physically and (b) any logical connection is not automated](https://csrc.nist.gov/glossary/term/air_gap), I think air-gapped Pawket satisfied all these requirements.

**Comment by wizicer** [comment]
Hi @SlowestTimelord , is the explanation acceptable? Can we move on to merge this PR?

**Comment by SlowestTimelord** [comment]
Thanks for the explanation. My issue is the feature term "Air-gapped" is not obvious what it means without seeing the explanation you've given. Can you think of a tooltip that can describe it succinctly? Or is there documentation or page that can be linked to? In its current form I feel like it would just raise more questions than it answers in wallet comparisons.

And just to check my understanding, the feature allows the separation of roles between an online and an airgapped device?
- Airgapped device: has the ability to sign tx
- Online device: has read-only ability on the live network
And communication between the two is done using QR codes to maintain the airgap?

**Comment by wizicer** [comment]
Your understanding is right.

And I add a tooltip like this:
image
The description above credits to [this link](https://supraoracles.com/academy/what-is-an-air-gapped-wallet/)

Is this acceptable?

File: labels/wallets.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h1>Wallets</h1>
              <th scope="col">NFT</th>
              <th scope="col">DID</th>
              <th scope="col">Offers</th>
-             <th scope="col">Memos</th>
+             <th scope="col"><abbr title="Air-gapped wallets are crypto wallets that are completely disconnected from the internet and any form of wireless communication. Providing additional security." class="featureQuestion">Air-gapped</abbr></th>
```
Comment (SlowestTimelord) [inline labels/wallets.html:28]: I'd suggest rewording this to:

> Air-gapped wallets have the option to be completely disconnected from the internet and any form of wireless communication, providing additional security.

As to not suggest Pawket is *only* an airgapped wallet. Otherwise LGTM.

**Review by SlowestTimelord — CHANGES_REQUESTED** [review]

**Review by SlowestTimelord — APPROVED** [review]
